include/dbus_utility.hpp

```diff
@@ -210,5 +214,19 @@ inline void
         "xyz.openbmc_project.ObjectMapper", "GetObject", path, interfaces);
 }
 
+inline void
+    getAssociationList(const std::string& service, const std::string& path,
```

Comment:
It can go upstream, right?

Reply:
I think it seems good to be in upstream.

Reply:
But we dont have code that calls  get with "xyz.openbmc_project.Association.Definitions", "Associations" in the upstream. So would they accept to introduce this method?